### PR TITLE
added 3 papers and link to code

### PR DESCRIPTION
**Pull request by arrenglover**
please move categories if required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -396,6 +398,9 @@ British Machine Vision Conf. (BMVC), London, 2017.
 - Delbruck, T., Villanueva, V., Longinotti, L.,  
 *[Integration of dynamic vision sensor with inertial measurement unit for electronically stabilized event-based vision](http://doi.org/10.1109/ISCAS.2014.6865714)*,  
 IEEE Int. Symp. Circuits and Systems (ISCAS) 2014, 2636-2639. [YouTube](https://youtu.be/Tzy4WF6Qp-Y)
+- Vasco, V., Glover, A., Mueggler, E., Scaramuzza, D., Natale, L., and Bartolozzi, C.  
+*[Independent Motion Detection with Event-driven Cameras](https://arxiv.org/abs/1706.08713v2)*,  
+IEEE/RAS Int. Conf. on Advanced Robotics (ICAR) 2017, Hong Kong, China.
```
Comment (guillermogb) [inline README.md]: This is a duplicate. Elias suggested section `Localization and Ego-Motion Estimation`. Should it be on both sections or just in one?

File: README.md [hunk 2]
```diff
@@ -449,7 +454,9 @@ Eur. Conf. Mobile Robots (ECMR), Lincoln, 2015. [PDF](http://rpg.ifi.uzh.ch/docs
 *[Human vs. computer slot car racing using an event and frame-based DAVIS vision sensor](https://doi.org/10.1109/ISCAS.2015.7169170)*,  
 IEEE Int. Symp. Circuits and Systems (ISCAS), Lisbon, 2015, pp. 2409-2412. [YouTube 1](https://youtu.be/CnGPGiZuFRI), [YouTube 2](https://youtu.be/ALneVn-Ls2Q)
 - [Moeys et. al. EBCCSP 2016](#Moeys16ebccsp).  *VISUALISE Predator/Prey Dataset*.
-
+- Vasco, V., Glover, A., Tirupachuri, Y., Solari, F., Chessa M. and Bartolozzi C.,
```
Comment (guillermogb) [inline README.md]: I tend to use "and" only when there are 2 authors.

File: README.md (hunk 1 above)
Comment (arrenglover) [inline README.md]: sorry - my search skills need improvement apparently ;) - I'll just leave it where it is.

File: README.md [hunk 3]
```diff
@@ -346,7 +346,7 @@ IEEE Int. Conf. Computational Photography (ICCP), Stanford, CA, USA, 2017, pp. 1
 *The Event-Camera Dataset and Simulator: Event-based Data for Pose Estimation, Visual Odometry, and SLAM.*
 - Vasco, V., Glover, A., Mueggler, E., Scaramuzza, D., Natale, L., Bartolozzi, C.  
 *Independent Motion Detection with Event-driven Cameras*,  
-Int. Conf. Advanced Robotics (ICAR), Hong Kong, 2017. [PDF](https://arxiv.org/pdf/1706.08713.pdf)
+Int. Conf. Advanced Robotics (ICAR), Hong Kong, 2017. [PDF](https://arxiv.org/pdf/1706.08713v2.pdf)
```
Comment (guillermogb) [inline README.md]: The link in the previous commit (without the "v2" suffix) links to the **latest** version of the paper. See for example [this one](https://arxiv.org/pdf/1610.08336.pdf), which should open v3.
I do not think that it matters for this paper, but it is good to know this trick :)

Comment (arrenglover) [inline README.md]: ah good to know - I haven't dealt with arxiv hosting before. Nice work with the repository.

Comment (guillermogb) [inline README.md]: Thanks!

**Pull request merged**